packages/cli/resources/inventory-report/welcome/welcome.hbs

```diff
@@ -142,9 +157,70 @@ and you don't see SQL queries here, you can add some test cases that interact wi
 the better the AppMap data will be for analyzing the SQL changes in each pull request.
 {{/if}}
 
+</details>
+
+<details>
+<summary>
+<h3><a id="large-appmaps">Large AppMaps</a></h3>
+</summary>
+
+Some tests can produce large AppMaps, because those they exercise longer or highly repeated code paths. 
+AppMap runtime code review is optimized for AppMaps that 
+are less than 10 MB in size. You may want to consider disabling AppMap for tests that produce really
+large AppMaps, since they can slow down your CI job and they are hard to analyze. Also, note that
```

Comment:
Saying that they're "hard to analyze" makes it sound like our product isn't very capable. Is there some other reason, preferably more specific, that we could use here?